contents/teams/support/objectives.mdx

```diff
@@ -1,19 +1,21 @@
-### Improve our SLA achievement without causing a CSAT catastrophe (Ben & Ben)
-- **Rationale:** Business as usual = A good start
-- **What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/246)
-- **We'll know we're successful when:** We're regularly hitting 90% SLA achievement for Normal+ prio tickets. CSAT doesn't fall below 80%.
+### Improve efficiency and consistency of how we use Zendesk (Ben)
+- **Rationale:** Less confusing Zendesk = more consistency across support and more efficient support.
+- **What we'll ship:** Improvements to, simplification of, and clarity around Zendesk procedures and implementation.
+- **We'll know we're successful when:** The the first resolution time and full resolution time metrics are going down.
 
-### Make everyone a subject matter expert as part of their onboarding (Abigail)
-- **Rationale:** More experts = fewer escalations
-- **What we'll ship:** Issue coming soon
-- **We'll know we're successful when:** Everyone is an expert in something and we have a playbook for learning when we hire next.
+### Streamline support workflow by implementing features in ZenHog (Haynes)
```

Comment:
My approval isn't needed, but as fly-by feedback: I adore this goal. Love that we're building our own tools and that things that originally came out of a hackathon can continue to improve and become more useful!